iModelCore/ecobjects/test/NonPublished/ECInstanceJsonWriterTests.cpp

```diff
@@ -441,46 +441,46 @@ TEST_F(ECInstanceJsonWriterTests, WriteEmbeddedStructValueTest)
                  }
               ]
            }
-        })");
+        })"));
 
     // Act
 
-    BeJsDocument actualJsonValue1;
+    BeJsDocument actualJsonValue1;

```

Comment:
Are there some unexpected or unintended line ending changes happening?  There are a lot of seemingly unchanged changed lines related to json.  Why are they being flagged as changed and is it intentional?

Reply:
Not sure. I will check…

Reply:
looks good